init/sysinit.go

```diff
@@ -132,6 +132,10 @@ func runContainersFrom(startFrom string, cfg *config.Config, containerConfigs []
 				log.Errorf("Failed to run %v: %v", containerConfig.Id, container.Err)
 			}
 
+			if cfg.Debug {
+				container.Log(os.Stdout, os.Stderr)
```

Comment:
As implemented right now, its not following. It just dumps the logs and exits, similar to the docker cli when there is a long running process.

For readability in the startup process, I'd like to make the call blocking. So I'd like to take out the go routine in the container.Log method. If we are going to keep the method as part of the Container type, I think we should add a boolean to follow, but not enable it here. 

Thoughts?